sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4365,8 +4365,15 @@ pi_result piEventRelease(pi_event Event) {
     die("piEventRelease: called on a destroyed event");
   }
 
+  // The event is no longer needed upstream, but we have to wait for its completion
```

Comment:
Looks like in adding this comment it may exceed line length limit?  clang-format check is failing.